app/javascript/components/GameDetails/ScoreBoard.jsx

```diff
@@ -8,63 +7,30 @@ import TableContainer from '@material-ui/core/TableContainer';
 import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
 import CardMedia from '@material-ui/core/CardMedia';
-
 import ScoreBoardStyles from './ScoreBoardStyles';
 import HammerIcon from '../../assets/hammer.svg';
 
-const useStyles = makeStyles(ScoreBoardStyles);
-
-function createTeamRow(name, score, teamId) {
-  return { name, score, teamId };
-}
-
-function createEndCell(end) {
-  return { end };
-}
 
-const ends = [
-  createEndCell(1),
-  createEndCell(2),
-  createEndCell(3),
-  createEndCell(4),
-  createEndCell(5),
-  createEndCell(6),
-  createEndCell(7),
-  createEndCell(8),
-  createEndCell(9),
-  createEndCell(10),
-  createEndCell(11),
-];
+const useStyles = makeStyles(ScoreBoardStyles);
 
 const ScoreBoard = ({ gameState }) => {
   const classes = useStyles();
-  const endsData = gameState.ends;
+  const { ends, currentEnd, teams_with_players } = gameState;
 
-  const teamOneId = gameState.teams_with_players[0].team.id;
-  const teamTwoId = gameState.teams_with_players[1].team.id;
+  const [endNumbers] = useState([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]);
```

Comment:
If this state is always just a array of the numbers 1 to 11 then why are we storing it as state? It will never change so it really doesn't need to be state. It could just be a constant outside of the component or we could even use a C style for loop intead of map later on.